frontend/projects/upgrade/src/assets/i18n/en.json

```diff
@@ -103,6 +103,7 @@
   "home.new-experiment.design.partition-id.placeholder.text": "ID",
   "home.new-experiment.design.add-experiment-partition.text": "Add Experiment Site",
   "home.new-experiment.design.assignment-weight-validation.text": "Assignment weights' sum must be 100",
+  "home.new-experiment.design.assignment-weight-negative.text": "Assignment weights' cannot be negative",
```

Comment:
@ppratikcr7 Can you update the value of text to either `Assignment weights' can not be negative` or `Assignment weights' can't be negative`?

Reply:
ok